src/bin/ccg.rs

```diff
@@ -0,0 +1,393 @@
+#[macro_use]
+extern crate log;
+
+use bob::configs::cluster::{Config, Node as ClusterNode, Replica, VDisk};
+use chrono::Local;
+use clap::{App, Arg, ArgMatches};
+use env_logger::fmt::Color;
+use log::{Level, LevelFilter};
+use std::fs::{File, OpenOptions};
+use std::io::{Read, Write};
+use std::sync::atomic::{AtomicUsize, Ordering};
+
+const ORD: Ordering = Ordering::Relaxed;
+
+#[tokio::main]
+async fn main() {
+    init_logger();
```

Comment:
--help with example how to execute would be nice

Reply:
clap provides help message with descrition of the each option